packages/core/core/src/services/document-service/utils/unidirectional-relations.ts

```diff
@@ -98,7 +96,7 @@ const sync = async (
       });
 
       // Insert those relations into the join table
-      await con.batchInsert(joinTable.name, newRelations).transacting(trx);
+      await trx.batchInsert(joinTable.name, newRelations, 1000);
```

Comment:
Is 1000 needed ? what if we have more ? should you add a check or sth ?

Reply:
👍

Reply:
https://knexjs.org/guide/utility.html#batchinsert

seems to be 1000 by default